README.md

```diff
@@ -0,0 +1,118 @@
+
+PyVista leverages [VTK](http://www.vtk.org) to provide a python-friendly API that is immediately usable without any prior knowledge of VTK and is being built as the 3D equivalent of [matplotlib](https://matplotlib.org/), with plugins to [Jupyter](https://jupyter.org/) to enable visualization of results using both server and client-side rendering.
+
+We will provide a hands-on tutorial accessible to anyone with internet access and a computer via open and accessible [example Jupyter notebooks](https://docs.pyvista.org/examples/index.html) through a comprehensive tutorial highlighting popular use cases.
+
+
+
+## Keywords
+
+[visualization](https://github.com/topics/visualization)
+[meshviewer](https://github.com/topics/meshviewer)
+[vtk](https://github.com/topics/vtk)
+[3d](https://github.com/topics/3d)
+
+## Other Information and Files
+See our examples at [PyVista Examples](http://docs.pyvista.org/examples/index.html)
+
+### Tutorial Description.
+
+- Use PyVista effectively to create 3D visualizations from a variety of datasets.
+- Understand the basic class and data structure of PyVista.
+- Be familiar of the various filters and features of PyVista.
+- Know which Python libraries which are used and can be used by PyVista.
+We see this tutorial catering to anyone who wants to visualize data in any domain, and this ranges from basic Python users to advanced power users.
+
+1. Basic knowledge of Python to get started. Be able to install jupyterlab on your machine and be up and running.
+2. Intermediate users will want to be familiar with [NumPy](https://numpy.org/) and other libraries that are compatible with PyVista, like `trimesh` or `meshio`.
+3. Advanced users will be familiar with GUI frameworks like Qt, VTK, and advanced data science.
+
+### Tutorial Outline.
+
+0. Getting Started - PyVista for 3D Visualization within Python. (10 min for talk, 10 min for exercise)
+
+1. Basic usage - Reading and plotting 3D data using built-in mesh data and external files.  (10 min for talk, 10 min for exercise)
+
+2. What is a Mesh? - Learn about the PyVista data types and how you can use it to visualize almost anything. (15 min for talk, 10 min for exercise)
+
+3. PyVista Data Model - Learn about the data model of the PyVista framework and how you can visualize your data in style using point, cell, and field data. (15 min for talk, 10 min for exercise)
+
+4. Break. Stretch fingers and grab some coffee. (15 minutes)
+
+5. Mesh animation - Show how to animate meshes using the PyVista API to make awesome online documentation. (10 min for talk, 10 min for exercise)
+
+6. PyVista, jupyterlab, and sphinx - Leverage PyVista to make some awesome interactive web documentation. (20 min for talk, 10 min for exercise)
```

Comment:
Perhaps 6 should focus on documentation and 5 animations/generating figures more generally?

Reply:
I'm trying to think of ways to make Section 5 better.
The Tutorial uses jupyterlab, so I agree with that as it is well explained in the other sections.
```suggestion
5. Mesh animation - Show how to animate meshes using the PyVista API to make awesome online documentation. (10 min for talk, 10 min for exercise)

6. PyVista and sphinx - Leverage PyVista to make some awesome interactive web documentation. (20 min for talk, 10 min for exercise)
```